src/lib/collections.js

```diff
@@ -276,6 +278,7 @@ var $builtinmodule = function (name) {
         }
 
         Sk.abstr.setUpInheritance("OrderedDict", mod.OrderedDict, Sk.builtin.dict);
+        mod.OrderedDict.tp$mro = new Sk.builtin.tuple([mod.OrderedDict, Sk.builtin.dict, Sk.builtin.object]);
```

Comment:
One question: Are `tp$mro`s now required for hand-crafted types like this? Because if so, this is probably going to be a breaking change for anyone who has taken these as examples for building their own classes.

(On the other hand, anyone who's copied this example is producing a purely prototypical class, so perhaps we're fine, and you're just setting these MROs for style points?)

Reply:
so I added these because in #1092's implementation of `$isSubType` , it assumed that every object had a `tp$mro`. In this branch when I used that approach it failed on something like:

```python
issubclass(OrderedDict, str) 
```

As a result I adjusted the `$isSubType` algorithm in this branch so that it wouldn't fail if `tp$mro` wasn't defined:
```javascript
    return this === other || 
           this.prototype instanceof other || 
          (this.hasOwnProperty("tp$mro") && this.tp$mro.v.includes(other));
```

So adding `tp$mro` to collections isn't necessary for this pr, but having noticed that the `tp$mro` didn't exist on `collection` classes I added them...